test/AutoCompleteTextField.spec.js

```diff
@@ -137,6 +137,12 @@ describe('option list filtering', () => {
     expect(component.find('.react-autocomplete-input > li')).to.have.length(2);
   });
 
+  it('ABC => 3 options', () => {
```

Comment:
Can we please have one more test case which is when typing lowercase 'abc' it should also return 3 results